include/image-commands.mk

```diff
@@ -344,3 +344,10 @@ define Build/kernel2minor
 	kernel2minor -k $@ -r $@.new $(1)
 	mv $@.new $@
 endef
+
+define Build/linksys-image
+	sh $(TOPDIR)/scripts/linksys-image.sh \
+		"$(call param_get_default,type,$(1),$(DEVICE_NAME))" \
```

Comment:
why is there and explicit sh call invoking a shell script ?

Reply:
It doesn't need to be explicit. It could just be called as $(TOPDIR)/scripts/linksys-image.sh.
Will remove 'sh'